cluster/plan.go

```diff
@@ -563,6 +568,29 @@ func (c *Cluster) BuildKubeletProcess(host *hosts.Host, serviceOptions v3.Kubern
 			fmt.Sprintf("%s:c:/host/run", path.Join(host.PrefixPath, "/run")),
 		}...)
 	} else {
+		// For Calico CNI fix
+		parsedVersion, err := getClusterVersion(c.Version)
+		if err != nil {
+			logrus.Debugf("Error while parsing cluster version: %s", err)
+		}
+		parsedSeptK8s125Version, err := getClusterVersion(SeptK8s125Version)
+		if err != nil {
+			logrus.Debugf("Error while parsing cluster version: %s", err)
+		}
+		parsedSeptK8s126Version, err := getClusterVersion(SeptK8s126Version)
+		if err != nil {
+			logrus.Debugf("Error while parsing cluster version: %s", err)
+		}
+		parsedSeptK8s127Version, err := getClusterVersion(SeptK8s127Version)
+		if err != nil {
+			logrus.Debugf("Error while parsing cluster version: %s", err)
+		}
+		if (parsedVersion.Major == parsedSeptK8s125Version.Major && parsedVersion.Minor == parsedSeptK8s125Version.Minor && parsedVersion.GE(parsedSeptK8s125Version)) ||
+			(parsedVersion.Major == parsedSeptK8s126Version.Major && parsedVersion.Minor == parsedSeptK8s126Version.Minor && parsedVersion.GE(parsedSeptK8s126Version)) ||
+			parsedVersion.GE(parsedSeptK8s127Version) {
+			Binds = append(Binds, "/var/log/calico/cni:/var/log/calico/cni:z")
```

Comment:
Just for clarification, adding calico logs have nothing to do with felix config right? This is an extra thing, or?